apps/web/app/constants.ts

```diff
@@ -8,8 +8,11 @@ export const PROTECTED_APP_URL_PATHS: RegExp[] = [
 	/^(\/profile(\/)?)(.*)$/,
 	/^(\/settings(\/)?)(.*)$/,
 	/^(\/task(\/)?)(.*)$/,
+	/^(\/call(\/)?)(.*)$/,
```

Comment:
@paradoxe35 I think here should be "meet" too instead of "call"?

Reply:
I fixed that myself @paradoxe35 , let me know if I did something wrong :D

Reply:
Yes @evereq 
I should be meet